LICENSE

```diff
@@ -0,0 +1,396 @@
+Copyright © 2020 WHATWG (Apple, Google, Mozilla, Microsoft).
```

Comment:
We don't include the year anymore. Is this file not a straight copy from another or is there another that still includes a year?

Reply:
I copied it over from my clone of the `whatwg.org` repo (and changed the date) — but I realize now that it was a stale copy; I guess I hadn’t pulled to update my clone of that repo in a long time.

Anyway, I now pulled in the `whatwg.org` repo and re-copied it over and amended the commit on this branch and re-pushed it here. So the date is gone now.